frontend/src/utils/offlineMessages.js

```diff
@@ -3,19 +3,27 @@
  */
 module.exports = {
```

Comment:
why this is called `module.exports`?

Reply:
pls see https://github.com/metasfresh/metasfresh/pull/12082#discussion_r764737620